### ISSUE TITLE
HLS WebGL video textures produces faulty colour rendering in IOS 10

### ISSUE THREAD

**Issue by danrossi** [mrdoob/three.js#10067]
I am separating this into a different ticket but they might be related to the FlipY bug #9754

I was able to successfully test the HLS webGL rendering on an Iphone after the FlipY work arounds that was figured out in the previous bug ticket. 

The simulator was causing similar colour faults, but when using the RGBA format flag, the output I see on the Iphone is the same regardless which RGB format is used. 

The colours are rendering incorrect. Using this code

```
gl.texImage2D(gl.TEXTURE_2D, 0, gl.RGB, gl.RGB, gl.UNSIGNED_BYTE, video);
```

If` there is a quick fix work around rather waiting months for webkit fixes that could be a start. 

So such HLS webgl output is only partially working on IOS 10 not IOS 9. Many devices are stuck on 9 now like my Ipad 3. I need to find a work around for 9.

I'm sorry if this has nothing to do with three.js and a webkit problem of course as standard. 

http://dev.electroteque.org/webgl/threejs-hls.html
http://dev.electroteque.org/webgl/webgl.html

**Comment by danrossi** [comment]
Refer to webkit ticket here https://bugs.webkit.org/show_bug.cgi?id=164540

**Comment by danrossi** [comment]
fixed in #9754
